src/Microsoft.AspNetCore.Sockets/Transports/LongPollingTransport.cs

```diff
@@ -35,7 +34,10 @@ public LongPollingTransport(IReadableChannel<Message> connection, ILoggerFactory
 
             try
             {
-                using (var message = await _connection.ReadAsync(_cancellationSource.Token))
+                // TODO: We need the ability to yield the connection without completing the channel.
```

Comment:
We won't be able to use this to indicate poll but don't complete the channel. We'll likely end up having to send a null message or something similar.